Test/comp/JsModule.dfy

```diff
@@ -1,6 +1,8 @@
-// RUN: %dafny /compile:3 "%s" /compileTarget:js > "%t"
-// note: putting /compileTarget:js after "%s" overrides user-provided option
-// RUN: %diff "%s.expect" "%t"
+/*
+---
+compile: 3
+compileTarget: js
```

Comment:
This seems like a nice and descriptive way to declare special parameters for those test cases that need it.

Reply:
Thanks, I agree! I really didn't want to invent anything new that had to parse myself. There are lots of examples of using JSON for data-driven tests but I find YAML cleaner and easier to read, and it's a superset of JSON anyway for more complicated cases.